native_client/javascript/package.json.in

```diff
@@ -37,6 +39,10 @@
       "node-wav": "0.0.2"
     },
     "devDependencies": {
+      "node-gyp": "4.x - 5.x",
```

Comment:
taking the opportunity to perform some cleanup: this will install `5.x` by default, but will keep `4.x` that we force-install on Windows.